suite-native/module-send-receive/src/components/TokenReceiveCard.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { AlertBox, Badge, Box, Text, VStack } from '@suite-native/atoms';
+import { EthereumTokenIcon } from '@trezor/icons';
+import {
+    EthereumTokenAmountFormatter,
+    EthereumTokenToFiatAmountFormatter,
+} from '@suite-native/formatters';
+import { getEthereumTokenIconName } from '@suite-native/ethereum-tokens';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { AccountKey, TokenAddress, TokenSymbol } from '@suite-common/wallet-types';
+import { AccountsRootState, selectAccountLabel } from '@suite-common/wallet-core';
+
+type TokenReceiveCardProps = {
+    tokenSymbol: TokenSymbol;
+    tokenName: string;
+    accountKey: AccountKey;
+    contract: TokenAddress;
+    balance?: string;
+};
+
+const tokenReceiveCardStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
+    padding: utils.spacings.medium,
+
+    borderRadius: utils.borders.radii.medium,
+}));
+
+const tokenDescriptionStyle = prepareNativeStyle(_ => ({
+    flexShrink: 1,
+}));
+
+const valuesContainerStyle = prepareNativeStyle(utils => ({
+    maxWidth: '40%',
+    flexShrink: 0,
+    alignItems: 'flex-end',
+    paddingLeft: utils.spacings.small,
+}));
+
+const iconContainerStyle = prepareNativeStyle(utils => ({
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 12,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: utils.colors.backgroundSurfaceElevation2,
+    marginRight: utils.spacings.medium,
+}));
+
+export const TokenReceiveCard = ({
```

Comment:
It was my intention to reuse TokenListItem. But the component differs in multiple places, so if I reused it, it would end up as a way longer file with many conditional rendering checks. For this reason, I decided to rather duplicate some code in the new TokenReceivedCard component and make it way readable that way.